README.md

```diff
@@ -48,8 +49,28 @@ CodeScene gives you a number of options that controls the scope of the delta ana
 
 The CodeScene API configuration section has to match the information specified inside CodeScene itself and retrievable from the analysis configuration.
 
+API Credentials should be added via [jenkins credentials plugin](https://wiki.jenkins-ci.org/display/JENKINS/Credentials+Plugin).
+Check [Injecting Secrets into Jenkins Build Jobs](https://support.cloudbees.com/hc/en-us/articles/203802500-Injecting-Secrets-into-Jenkins-Build-Jobs) for more details.
+
 ## Changelog
 
+* 1.1.0
+  - Use [credentials plugin](https://wiki.jenkins-ci.org/display/JENKINS/Credentials+Plugin)
+    for storing CodeScene API credentials instead of hardcoded username and password.
+    Old jobs will continue to work, but you won't be able to update job configuration
+    (e.g. change username or password) without using the new way for credentials configuration.
```

Comment:
"without using the new way for credentials configuration" => "until you switch to the credentials plugin"